lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -20,6 +20,9 @@
         <tr>
           <td>${_("Honor")}</td><td>${modes['honor']}</td>
         </tr>
+        <tr>
+          <td>${_("Professional")}</td><td>${modes['professional'] + modes['no-id-professional']}</td>
```

Comment:
I made the decision to aggregate this at the presentation layer rather than in the CourseEnrollment model's classmethod.  It's not great that we need to do this kind of aggregation, but this way at least the backend code consistently treats these as two separate course modes (which is what they are).